apps/api/src/persistence/database/utilities/mapEdgeConnectionMembersToArangoDocumentDirectionAttributes.ts

```diff
@@ -0,0 +1,45 @@
+import {
+    EdgeConnectionMember,
+    EdgeConnectionType,
+} from 'apps/api/src/domain/models/context/edge-connection.entity';
+import { ResourceCompositeIdentifier } from 'apps/api/src/domain/models/types/entityCompositeIdentifier';
+import { InternalError } from 'apps/api/src/lib/errors/InternalError';
+import { PartialDTO } from 'apps/api/src/types/partial-dto';
+import { HasArangoDocumentDirectionAttributes } from '../types/HasArangoDocumentDirectionAttributes';
+import convertResourceCompositeIdentifierToArangoDocumentHandle from './convertResourceCompositeIdentifierToArangoDocumentHandle';
+
+/**
+ * For an edge document, Arango requires two `special attributes` `to` and `from`
+ */
+export default (
```

Comment:
Rename this helper to `getArangoDocumentDirectionAttributesFromEdgeConnectionMembers`

Reply:
done